Automattic/crowdsignal-plugin
# Add icon and links for projects

Fixes  c/BODS89tU-tr

#### Changes proposed in this Pull Request:

* add an icon and correct links to edit/results for the Project type
* add a "generic" option at the end of the elseif statements for any types that don't exist yet.

#### Testing instructions:

* Make sure you have a project in your CS account, update the plugin and look at the CS dashboard in WP admin.  Verify the Project type has its own icon.  Verify the links for results and edit go to the right place.


#### Screenshot / Video




#### Proposed changelog entry for your changes:
Add support for Projects in dashboard view
